generate-profile.lic

```diff
@@ -0,0 +1,120 @@
+=begin
```

Comment:
In the past we've used the `/util` directory for these kinds of "doesn't interact with the game" scripts... but I'm not opposed to leaving it here, either.

Reply:
If this were moved to `/util`, what would the command be to run the script?  `;util/gen`?

Reply:
well util/* isn't shipped with dr-scripts unless its a git checkout, so...

Reply:
Oh, right. Good point - that depends on the target users for this.